crates/cxx-qt-build/src/lib.rs

```diff
@@ -498,6 +498,25 @@ impl CxxQtBuilder {
             qtbuild.version().major.to_string(),
         );
 
+        for minor in 0..=qtbuild.version().minor {
+            let at_least_qt_version = format!(
+                "cxxqt_at_least_qt_version_{}_{}",
```

Comment:
Overall I like the `at_least`, it's nice and explicit :)

I'd prefer, if it was `qt_version_at_least`, not `at_least_qt_version` though, as that would be in line with `qt_version_major` and `qt_version_minor`.